WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift

```diff
@@ -0,0 +1,140 @@
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.heightAnchor.constraint(greaterThanOrEqualToConstant: 40).isActive = true
+        button.setTitleColor(UIColor.primary, for: .normal)
+        button.titleLabel?.font = WPStyleGuide.fontForTextStyle(.subheadline)
+        button.titleLabel?.adjustsFontForContentSizeCategory = true
+        button.titleLabel?.adjustsFontSizeToFitWidth = true
+        button.contentHorizontalAlignment = .leading
+        button.addTarget(self, action: #selector(buttonShowMoreTapped), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var contentStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.layoutMargins = UIEdgeInsets(top: 16, left: 16, bottom: 0, right: 16)
+        stackView.spacing = 4
+        stackView.isLayoutMarginsRelativeArrangement = true
+        return stackView
+    }()
+
+    private var blog: Blog?
+    private weak var presentingViewController: BlogDashboardViewController?
+
+    // MARK: - Initializers
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - View setup
+
+    private func setupView() {
+        contentView.addSubview(frameView)
+        frameView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.pinSubviewToAllEdges(frameView, priority: UILayoutPriority(999))
+
+        frameView.add(subview: contentStackView)
+        contentStackView.addArrangedSubview(campaignView)
+        contentStackView.addArrangedSubview(buttonShowMoreCampaigns)
+    }
+
+    // MARK: - BlogDashboardCardConfigurable
+
+    func configure(blog: Blog, viewController: BlogDashboardViewController?, apiResponse: BlogDashboardRemoteEntity?) {
+        self.blog = blog
+        self.presentingViewController = viewController
+
+
+        frameView.setTitle(Strings.cardTitle)
+
+        let viewModel = DashboardBlazeCampaignCardCellViewModel(response: mockResponse)!
+        campaignView.configure(with: viewModel.campaign, blog: blog)
+        buttonShowMoreCampaigns.isHidden = viewModel.isButtonShowMoreHidden
+        buttonShowMoreCampaigns.setTitle(viewModel.buttonShowMoreTitle, for: .normal)
+    }
+
+    // MARK: - Actions
+
+    @objc private func buttonShowMoreTapped() {
+        // TODO: Show campaign list
+    }
+}
+
+private extension DashboardBlazeCampaignCardCell {
+    enum Strings {
+        static let cardTitle = NSLocalizedString("my-sites.blazeCampaigns.title", value: "Blaze campaign", comment: "Title for the card displaying blaze campaigns.")
+
+        static func buttonShowMoreText(forCampaignCount count: Int) -> String {
+            let format = count == 1 ? showMoreCampaignsOne : showMoreCampaignsTwoOrMore
```

Comment:
It looks like [plurals are not supported](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/docs/localization.md#pluralization), so I followed the recommended approach.

Reply:
It looks like this button will probably need to go based on the internal discussion.